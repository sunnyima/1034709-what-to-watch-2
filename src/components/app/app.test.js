import React from 'react';
import renderer from 'react-test-renderer';
import App from '../app/app';

it(`App correctly renders after relaunch`, () => {
  const tree = renderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
});
