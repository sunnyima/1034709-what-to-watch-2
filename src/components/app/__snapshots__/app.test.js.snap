// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after relaunch 1`] = `
<div>
  <div>
    <p>
      Fantastic Beasts
    </p>
  </div>
  <div>
    <p>
      Bohemian Rhapsody
    </p>
  </div>
  <div>
    <p>
      Macbeth
    </p>
  </div>
</div>
`;
